react-app/src/app/screens/User/index.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { NavLink, Redirect, Route, Switch } from 'react-router-dom';
+
+import style from './styles.scss';
+import Dashboard from './components/Dashboard';
+import History from './components/History';
+import Points from './components/Points';
+
+class Userpage extends Component {
+  render() {
+    return (
+      <div className={style.loginLayout}>
+        <h2>User page</h2>
+        <div className={style.userbarNav}>
+          <NavLink to="/user" exact className={style.userbarLink} activeClassName={style.userbarActive}>
```

Comment:
Notice that these 4 `NavLink`s are the same, but with different link and text. You can extract them as an array and map them here.